jest.config.base.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   transform: {
     '^.+\\.ts$': 'ts-jest',
   },
+  setupFiles: ['dotenv/config'],
```

Comment:
Oh also, what is this for?

Reply:
Jest doesn't support passing env variables through commands like `DEBUG=true; yarn test`
So to pass through any env variable we should set up an `.env` file on the same directory where `jest.config.js` is.

Reply:
Hm, I’ve been using inline variables without a problem. Can you try removing the semicolon in between? e.g. `DEBUG=true yarn test`

Reply:
oh yeah removing semicolon works well!